packages/clay-button/src/Group.tsx

```diff
@@ -31,18 +31,15 @@ const ClayButtonGroup = ({
 		{...otherProps}
 		className={classNames(
 			className,
-			vertical ? 'btn-group-vertical' : 'btn-group'
+			vertical
+				? 'btn-group-vertical'
+				: spaced
+				? 'btn-group-spaced'
+				: 'btn-group'
```

Comment:
I think this is better for reading.

```suggestion
			{
						'btn-group-vertical': vertical,
						'btn-group-spaced': spaced,
						'btn-group': !spaced && !vertical,
			}
```